precompiles/werc20/integration_test.go

```diff
@@ -93,19 +100,8 @@ var _ = Describe("WEVMOS Extension -", func() {
 
 	Context("WEVMOS specific functions", func() {
 		When("calling deposit correctly", func() {
-			It("should emit the Deposit event but not modify the balance", func() {
-				depositCheck := passCheck.WithExpPass(true).WithExpEvents(werc20.EventTypeDeposit)
-				txArgs, callArgs := s.getTxAndCallArgs(erc20Call, contractData, werc20.DepositMethod)
-				txArgs.Amount = amount
-
-				_, _, err := s.factory.CallContractAndCheckLogs(sender.Priv, txArgs, callArgs, depositCheck)
-				Expect(err).ToNot(HaveOccurred(), "unexpected result calling contract")
```

Comment:
I think rather than deleting this we could check that there are indeed zero events